src/main/kotlin/org/rust/ide/colors/RustColorSettingsPage.kt

```diff
@@ -10,7 +10,13 @@ class RustColorSettingsPage : ColorSettingsPage {
     private val ATTRS = RustColor.values().map { it.attributesDescriptor }.toTypedArray()
 
     // This tags should be kept in sync with RustHighlightingAnnotator highlighting logic
+    // @TODO generate from RustColor
+    // @TODO Figure out how to namespace menu elements a la
+    //       https://github.com/JetBrains/kotlin/blob/8b30e7ef4e48494ba245b441a5b23142f1d6ae33/idea/idea-analysis/src/org/jetbrains/kotlin/idea/KotlinBundle.properties
```

Comment:
I don't know whether this is available on all IDEs we support. @matklad haven't these groups been introduced in 2016.1, have they?